base/src/main/java/com/mindera/skeletoid/threads/utils/ThreadUtils.kt

```diff
@@ -0,0 +1,13 @@
+package com.mindera.skeletoid.threads.utils
+
+object ThreadUtils {
+
+    /**
+     * Gets the current thread name
+     *
+     * @return The current thread name in a printable string
+     */
+    fun getCurrentThreadName() : String{
```

Comment:
if you add the notation `@JvmStatic` you can remove the need to use all those ugly `INSTANCE` accesses from Java